package/yast2-cluster.spec

```diff
@@ -36,6 +36,7 @@ BuildRequires:  firewall-macros
 BuildArch:      noarch
 
 # SuSEFirewall2 replaced by Firewalld (fate#323460)
+Requires:       sharutils
```

Comment:
You introduced the new "Requires" between the yast2 Requires and its comment. Now it looks like the comment in the previous line refers to sharutils instead of yast2. Confusing. Please fix it.

Reply:
The comment is mainly about an outdate change. I will remove it.